github-lite/tests/test_github.py

```
@@ -0,0 +1,131 @@
+
+
+def test_parse_overrides():
+    raw = "work/app=acme/app, work/lib=acme/lib"
+    assert parse_overrides(raw) == {"work/app": "acme/app", "work/lib": "acme/lib"}
+
+
+def test_parse_overrides_ignores_invalid():
+    assert parse_overrides("nonsense, a=b") == {"a": "b"}
+
+
+def _mock_response(data: list[dict[str, Any]], headers: dict[str, str] | None = None) -> mock.Mock:
```

Ruff (0.16.1)

[error] Undefined name `Any`

(F821)